cloud/shared/bin/lib/setup_template.py

```diff
@@ -34,12 +33,6 @@ def setup_log_file(self):
         _, self.log_file_path = tempfile.mkstemp()
         print(" - TODO: Setup log file here.")
 
-    def _make_backend_override(self):
-        current_directory = self.config.get_template_dir()
-        shutil.copy2(
```

Comment:
Was removal of this code necessary? Or was it unused?

Reply:
It's a duplicate of the same function 20 lines above.